ws_sdk/ws_utilities.py

```diff
@@ -380,8 +381,10 @@ def get_latest_ua_release_version() -> str:
 
 def get_latest_ua_release_url() -> dict:
     res = call_gh_api(url=LATEST_UA_URL)
-
-    return json.loads(res.text)
+    if res:
+        return json.loads(res.text)
+    else:
+        return {}
```

Comment:
returning a blank object will have cascading effects on the methods calling this. Can you ensure the workflow handles this return? See get_latest_ua_release_version @ line 375

Reply:
In this case, it will return the standard message "an invalid version and will not be supported in a future release"